lib/boorus/danbooru/presentation/features/accounts/login/widgets/login_box_widget.dart

```diff
@@ -213,13 +213,13 @@ class LoginField extends HookWidget {
 
   @override
   Widget build(BuildContext context) {
-    final _controller = useTextEditingController();
+    final controller = useTextEditingController();
 
     return TextFormField(
       onChanged: onChanged,
       obscureText: obscureText,
       validator: validator,
-      controller: controller ?? _controller,
```

Comment:
if you remove the null check here and rename the controller, please remove this too and make the controller non-nullable.
https://github.com/khoadng/Boorusama/blob/ff79645091738c086e43daf736714f2aec137067/lib/boorus/danbooru/presentation/features/accounts/login/widgets/login_box_widget.dart#L216